contents/terms.mdx

```diff
@@ -108,7 +108,7 @@ By signing up for PostHog Scale License, you and any entity that you represent (
 
 2.3 Customer will be responsible for maintaining the security of Customer’s account, passwords (including but not limited to administrative and User passwords) and files, and for all uses of Customer account with or without Customer’s knowledge or consent.
 
-2.4 Customer will not air gap their PostHog Scale instance without prior permission from PostHog, as this prevents us from being able to bill you correctly based on usage. Flat usage billing is only available under the PostHog Enterprise License.
+2.4 Customer will not air gap their PostHog Self Host instance without prior permission from PostHog, as this prevents us from being able to bill you correctly based on usage. Flat usage billing is only available under the PostHog Enterprise License.
 
```

Comment:
De we need to remove
> Flat usage billing is only available under the PostHog Enterprise License
And handle it separately under an MSA?  This rarely comes up but now we're saying there's no such thing as an Enterprise license

Reply:
Good point - I can just remove the reference to flat billing entirely? Seems like something that a large customer will proactively ask for, rather than something that needs to be in our terms anyway, what do you think?